buildenv.mk

```diff
@@ -89,7 +89,7 @@ COMMON_FLAGS += -Wall -Wextra -Winit-self -Wpointer-arith -Wreturn-type \
 CFLAGS += -Wjump-misses-init -Wstrict-prototypes -Wunsuffixed-float-constants
 
 # additional warnings flags for C++
-CXXFLAGS += -Wnon-virtual-dtor
+CXXFLAGS += -Wnon-virtual-dtor -DPROTOBUF_INLINE_NOT_IN_HEADERS=0
```

Comment:
This is needed to prevent some warnings created by protobuf. See here for more infos https://stackoverflow.com/questions/34474091/protobuf-inline-not-in-headers#34485004